tests/test_setups/test_setup_functions.py

```diff
@@ -87,6 +87,6 @@ def test_setup_get(self):
             current = openml.setups.get_setup(setups[idx])
             assert current.flow_id > 0
             if num_params[idx] == 0:
-                self.asserts(current.parameters is None)
+                self.assertTrue(current.parameters is None)
```

Comment:
Sorry, I should have been more specific. The correct one here would be `self.assertIsNone`.